### ISSUE TITLE
Easy way to load all needed packages after activating enviroment?

### ISSUE THREAD

**Issue by oxinabox** [invenia/JLSO.jl#94]
The enviroment stored in the JLSO means we always have access to the right packages.
Bu they need to be `using`ed (or at least `Base.require`ed) to have the JLSO correctly deserialize any types from them.
Else you get a `UInt8` array.

Ideally we would workout which are actually used in the JLSO for a particular object and be able to `using` them.

A work around is to load everything.
It is a bit gross but the following works (I think it needs julia 1.4)

```julia
using Pkg: Pkg
eval(Expr(:using, (Expr(:., n) for n in Symbol.(keys(Pkg.Types.Context().env.project.deps)))...))
```

**Comment by axdahl** [comment]
I just tested in Julia 1.5.3 and it worked.

**Comment by rofinn** [comment]
Yeah, but unless we're inspecting the serialized objects this feels like more of a Pkg issue?

**Comment by oxinabox** [comment]
> Yeah, but unless we're inspecting the serialized objects this feels like more of a Pkg issue?

I think we should do exactly that.

**Comment by rofinn** [comment]
Hmm, maybe we should do that in another package? I've heard people complain that JLD2.jl doesn't do that either.

**Comment by oxinabox** [comment]
I think we should work it out here, and worry about generalizing later

**Comment by rofinn** [comment]
Okay, added a MWE in the PR linked.
